jena-arq/src/main/java/org/apache/jena/riot/system/RiotLib.java

```diff
@@ -73,12 +78,37 @@ public static Node createIRIorBNode(String iri)
         return NodeFactory.createURI(iri) ;
     }
 
-    /** Test whether  */
-    public static boolean isBNodeIRI(String iri)
-    {
+    /** Test whether a IRI is a ARQ-encoded blank node. */
+    public static boolean isBNodeIRI(String iri) {
         return skolomizedBNodes && iri.startsWith(bNodeLabelStart) ;
     }
     
+    // These two must be in-step.
+    /** Function applied to undefined prefixes to convert to a URI string */  
+    public static final Function<String,String> fixupPrefixes      = (x) -> "::"+x ;
```

Comment:
This could be
```
private static final String URI_SKELETON = "::";
...
public static final Function<String,String> fixupPrefixes = URI_SKELETON::concat;
```
I find that clearer, because it names the string, but then, my tastes are odd.